scripts/exrrfs_save_fcst.sh

```diff
@@ -26,13 +18,8 @@ echo "forecast length for this cycle is ${fcst_len_hrs_thiscyc}"
 #
 # decide the location of run and umbrella
 #
-if [[ -z "${ENS_INDEX}" ]]; then
-  ensindexstr=""
-else
-  ensindexstr="/mem${ENS_INDEX}"
-fi
-workdir="${DATAROOT}${ensindexstr}/${RUN}_fcst_${cyc}"
-umbrelladir="${UMBRELLA_DATA}/${jobid}"
+workdir="${DATAROOT}/${RUN}_fcst${MEMID}_${cyc}"
+umbrelladir="${UMBRELLA_DATA}${MEMDIR}/${jobid}"
```

Comment:
Similarly as above, `UMBRELLA_DATA` should be `UMBRELLA_ROOT` and `umbrelladir` should be `UMBRELLA_DATA` which should be defined in the J-Job.

Reply:
There is no UMBRELLA_ROOT. UMBRELLA space is in DATAROOT. Need to think about how to use the umbrella more. We can leave it as is for now.